releasetasks/templates/firefox/publish_balrog.yml.tmpl

```diff
@@ -0,0 +1,44 @@
+{% set buildername = "release-{}-{}_publish_balrog".format(branch, product) %}
+
+-
+    taskId: "{{ stableSlugId(buildername) }}"
+    requires:
+        - "{{ stableSlugId('publish_release_human_decision') }}"
+    reruns: 5
+    task:
+        provisionerId: "buildbot-bridge"
+        workerType: "buildbot-bridge"
+        created: "{{ now }}"
+        deadline: "{{ now.replace(days=4) }}"
+        expires: "{{ never }}"
+        priority: "high"
+        retries: 5
+        scopes:
+            - project:releng:buildbot-bridge:builder-name:{{ buildername }}
+        routes:
+            - index.releases.v1.{{ branch }}.{{ revision }}.{{ product }}.{{ version | replace(".", "_") }}.build{{ buildNumber }}.publish_balrog
+            - index.releases.v1.{{ branch }}.latest.{{ product }}.latest.publish_balrog
+        payload:
+            buildername: "{{ buildername }}"
+            sourcestamp:
+                branch: "{{ repo_path }}"
+                revision: "{{ mozharness_changeset }}"
+            properties:
```

Comment:
having not looked at publish to balrog mh script, i'll just trust that these props are correct :)